### PR TITLE
APP-332: Check "Objective C Endpoint Ownership Demo" and include it to sandbox

### PR DESCRIPTION
**Pull request by am1kk**
Don't merge.

### CODE REVIEW

File: verifiersdemo/resources/projects.xml [hunk 1]
```diff
@@ -57,357 +57,392 @@ For more background on the user verification refer to the corresponding design r
         <platforms>ANDROID</platforms>
         <features>EVENT USER_VERIFIER</features>
         <complexity>BASIC</complexity>
+        <bundleId>verifiers_demos</bundleId>
```
Comment (Slash32) [inline verifiersdemo/resources/projects.xml:60]: Why is the bundleId dropped here?

File: verifiersdemo/resources/projects.xml [hunk 2]
```diff
@@ -57,357 +57,392 @@ For more background on the user verification refer to the corresponding design r
...
+        <sdkLanguage>JAVA</sdkLanguage>
+        <platforms>LINUX_X86 WINDOWS_X86</platforms>
         <features>USER_VERIFIER</features>
-        <complexity>BASIC</complexity>
-        <bundleId>verifiers_demo</bundleId>
-        <sourceArchive></sourceArchive>
-        <projectFolder></projectFolder>
-        <sdkLibDir></sdkLibDir>
-        <destBinaryFile></destBinaryFile>
+        <complexity>REGULAR</complexity>
```
Comment (Slash32) [inline verifiersdemo/resources/projects.xml:78]: This application is of BASIC complexity type.

**Review by Slash32 — CHANGES_REQUESTED** [review]
There are questions to the PR that must be resolved before merge.

**Comment by Oldnickus** [comment]
@kaa-leeroy test this hard please

**Comment by Oldnickus** [comment]
@kaa-leeroy test this hard please

**Comment by kaa-leeroy** [comment]
Can one of the admins verify this patch?

**Comment by Oldnickus** [comment]
@kaa-leeroy test this hard please

**Comment by Oldnickus** [comment]
@kaa-leeroy test this hard please

**Comment by Oldnickus** [comment]
@kaa-leeroy test this hard please

**Comment by Oldnickus** [comment]
@kaa-leeroy test this hard please

**Comment by Oldnickus** [comment]
@kaa-leeroy test this hard please

**Comment by kaa-leeroy** [comment]
Can one of the admins verify this patch?

**Comment by Oldnickus** [comment]
@kaa-leeroy test this hard please

**Comment by Oldnickus** [comment]
@kaa-leeroy test this hard please

**Comment by Oldnickus** [comment]
@kaa-leeroy test this hard please

**Comment by Oldnickus** [comment]
@kaa-leeroy test this hard please

**Comment by Oldnickus** [comment]
@kaa-leeroy test this hard please

**Comment by Oldnickus** [comment]
@kaa-leeroy test this hard please

**Comment by Oldnickus** [comment]
@kaa-leeroy test this hard please

**Comment by Slash32** [comment]
@kaa-leeroy test this hard please

**Comment by Slash32** [comment]
@kaa-leeroy test this hard please

**Comment by Slash32** [comment]
@kaa-leeroy test this hard please

**Comment by Slash32** [comment]
@kaa-leeroy test this hard please

**Review by Slash32 — APPROVED** [review]

**Pull request merged**